CordovaDemo/platforms/ios/CordovaDemo/Plugins/com.pspdfkit.cordovaplugin/PSPDFKitPlugin.m

```diff
@@ -967,6 +967,18 @@ - (NSNumber *)shouldAskForAnnotationUsernameAsJSON
     return @(_pdfController.configuration.shouldAskForAnnotationUsername);
 }
 
+- (void)setPageGrabberEnabledForPSPDFViewControllerWithJSON:(NSNumber *)pageGrabberEnabled
+{
+    [_pdfController updateConfigurationWithBuilder:^(PSPDFConfigurationBuilder *builder) {
+        builder.pageGrabberEnabled = pageGrabberEnabled.boolValue;
+    }];
+}
+
+- (NSNumber *)pageGrabberEnabledAsJSON
+{
+    return @([_pdfController.configuration isPageGrabberEnabled]);
```

Comment:
I actually thought about it, I opened for the current syntax because we had a getter for the property. 

That said, I fully agree. Let's aim for consistency. 👏 

Addressed!